src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Text.txt

```diff
@@ -81,4 +72,16 @@ Blank()
 "B"
 
 >> Char(1/0)
-#Error
\ No newline at end of file
+#Error
+
+>> Text(DateTimeValue("03/15/2018 12:00:00"), "yyyy-MM-ddTHH:mm:ss")
```

Comment:
Similar to the comment below. We should use something like `Date(2018, 3, 15) + Time(12, 0, 0)`; check with @bryancroteau-MSFT if this is already implemented, though.

Reply:
This is implemented in LibraryOperators.cs

Reply:
I've removed this test as I added it to check something I need for a specific purpose and having the test re-written as shown doesn't really prove what I needed so the purpose of the test is lost.